test/general/fork.pass.cpp

```diff
@@ -0,0 +1,27 @@
+#include <oneapi/dpl/execution>
```

Comment:
Do we need a header copyright comment for this test, or is it unnecessary?

Reply:
We do need some header, thanks for reminding.